bin/cli.mjs

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+import { Command, Option } from 'commander';
+import { argv } from 'node:process';
+import { resolve } from 'node:path';
+import createGenerator from '../src/generators.mjs';
+import createLoader from '../src/loader.mjs';
+import createParser from '../src/parser.mjs';
+import { CLI_TARGET_MAPPING } from '../src/constants.mjs';
+
+const program = new Command();
+
+program
+  .requiredOption(
+    '-i, --input <patterns...>',
+    'Specify input file patterns using glob syntax'
+  )
+  .requiredOption('-o, --output <path>', 'Specify the output directory path')
+  .addOption(
+    new Option('-t, --target <mode>', 'Set the processing target mode').choices(
```

Comment:
What does this accept? `-t html,json`? for example?

Reply:
You can use the following format; `-t html json`